injector_test.py

```diff
@@ -62,12 +63,12 @@ def __init__(self, b: EmptyClass):
         self.b = b
 
 
-def prepare_nested_injectors():
+def prepare_nested_injectors(unique=False):
     def configure(binder):
         binder.bind(str, to='asd')
 
     parent = Injector(configure)
-    child = parent.create_child_injector()
+    child = parent.create_child_injector(unique=unique)
```

Comment:
Wouldn't it be even more interesting to add the `unique` flag to the parent injector as well, to see that they interact as expected, since it's an even stricter case? I think that it should be allowed and work with the current implementation.

Reply:
Yup, I agree. Also: I'm wondering if child injectors shouldn't inherit the `unique` value of the parent injector by default.

Reply:
That crossed my mind as well, and yes, I think that would be reasonable.